server.go

```diff
@@ -108,9 +108,17 @@ func mainHandler(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func adminHandler(w http.ResponseWriter, r *http.Request) {
+	path := r.URL.Path[1:]
+	log.Printf("Serving Admin file: %s", path)
+	appPath := fmt.Sprintf("%s", path)
```

Comment:
you do not need to use this variable, you an use path directly

Reply:
Sure! becase there isn't some text wich to chain to path

Reply:
To be precisse, you can just do:
```javascript
	path := r.URL.Path[1:]
	log.Printf("Serving Admin file: %s", path)
        http.ServeFile(w, r, path)
```